microsoft-edge/devtools-guide-chromium/network/index.md

```diff
@@ -32,28 +33,33 @@ For an overview of the network-related DevTools features, see [Network features
 
 <!-- ------------------------------ -->
 #### Video: Learn to Use the Network Tool
+<!-- equiv. to top of upstream -->
 
 For a video version of this tutorial, view the following video:
 
 [![Thumbnail image for video "Learn to Use the Network Tool"](./index-images/network-tool-intro-video.png)](https://www.youtube.com/watch?v=niG_Ck6E5L8)
 
 
-<!-- TODO: entire section needs a Microsoft Edge DevTools rewrite  -->
```

Comment:
removed TODO.  unclear which "entire section", presumably the one h2 section below.
Changed mid-sentence list to unordered list.
@captainbrosset

Reply:
Sure, that's ok.